bluetail/tests/test_scripts.py

```diff
@@ -0,0 +1,28 @@
+import os
+import subprocess
+from unittest import TestCase
+from django.conf import settings
+
+SCRIPTS_DIR = os.path.join(settings.BASE_DIR, "script")
+
+
+class TestFlagHelperFunctions(TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_script_setup(self):
+        db = settings.DATABASES["default"]
+        test_db_url = "postgres://{USER}:{PASSWORD}@{HOST}:{PORT}/{NAME}".format(
+            USER=db["USER"],
+            PASSWORD=db["PASSWORD"],
+            HOST=db["HOST"],
+            PORT=db["PORT"],
+            NAME=db["NAME"],
+        )
+        my_env = os.environ.copy()
+        my_env["DATABASE_URL"] = test_db_url
+
+        setup_script_path = os.path.join(SCRIPTS_DIR, "setup")
+
+        subprocess.call(setup_script_path, env=my_env, stderr=subprocess.PIPE, stdout=subprocess.PIPE)
```

Comment:
If you're mainly interested in testing the `import_example_data` management command then you can do `from django.core.management import call_command`, which then makes this a one-liner!

```python
call_command('insert_example_data')
```

More info in the docs - https://docs.djangoproject.com/en/2.2/topics/testing/tools/#management-commands

Reply:
@chrismytton Thanks for the review and feedback, great to get fresh eyes on this, although I should have labelled this as a draft before so it was clear we were still working.
We should get the data issues sorted today. ;) 

> If you're mainly interested in testing the `import_example_data` management command then you can do `from django.core.management import call_command`, which then makes this a one-liner!

Good point, thanks for suggestion. 
I created this test to help debug issues running scripts in Vagrant and local shells which is why I initially went for subprocess, but maybe it's good to test the command separately as well. 

FYI, so far we've not been in the mindset of creating comprehensive tests as this is only meant to be an Alpha and I haven't wanted to get too caught up in spending time making good tests before having a working prototype and knowing how things will be structured. When I'm debugging or trying to solve issues though I have switched to TDD and using tests to aid dev locally, but haven't tested in CI.

Reply:
Yep totally agree, data issues are the priority 🙂 

Understand what you're saying about testing. Aiming for comprehensive testing isn't appropriate for this stage of the project.